test/cases/testdata/containskeyword/test-contains-future-keyword.yaml

```diff
@@ -0,0 +1,67 @@
+cases:
```

Comment:
ℹ️ The main purpose of this file is to assert that everything goes well when using the `contains` builtin. While `if` is a purely superficial addition -- not changing anything related to how stuff is evaluated -- we could potentially mess up things when it comes to the `contains` builtin.